### PR TITLE
Add requirement to send mail to Eiffel Community maillist

### PR DESCRIPTION
**Pull request by fdegir**
New project proposals and proposals to move projects to different
lifecycle stages should be announced on Eiffel Community maillist
in order to increase the visibility.

Once the project creation or project lifecycle process is concluded,
TC chair sends an email to Eiffel Community maillist and announce
the new project or the lifecycle stage to broader Eiffel Community.

### Applicable Issues

Closes AP from [TC meeting](https://hackmd.io/SCImga0nS1qSh3QvsEOAVQ?view#Action-Items).

### Description of the Change

Project proposals and lifecycle stage changes are handled via GitHub issues in Eiffel Community community repository, which is not monitored by broader community.
In order to increase the visibility, transparency and community participation, it is important to announce this type of changes to the community on Eiffel Community Maillist.

This commit updates project lifecycle document to require such mails to be sent to the maillist.

### Alternate Designs

N/A

### Benefits

Increased visibility, transparency, and community participation.

### Possible Drawbacks

N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fatih Degirmenci fatih.degirmenci@est.tech

### CODE REVIEW

**Review by e-backmark-ericsson — APPROVED** [review]

**Review by sodero — APPROVED** [review]

File: PROJECT_LIFECYCLE.md [hunk 1]
```diff
@@ -38,12 +38,19 @@ their ability.
 - names of initial maintainers
 - infrastructure needs such as Git repositories
 
-Regarding project naming conventions, the Eiffel Community enforces no name convention but meaningful descriptive 
-names are encouraged. 
+In addition to formally proposing the project on GitHub, community members proposing the new project are expected
+to announce the project proposal on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) by
+sending an email in order to increase the visibility. The email should contain same information as the proposal
```
Comment (d-stahl-ericsson) [inline PROJECT_LIFECYCLE.md:43]:
```suggestion
sending an email in order to increase the visibility. The email should contain the same information as the proposal
```

File: PROJECT_LIFECYCLE.md [hunk 2]
```diff
@@ -57,6 +64,9 @@ Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) by sendi
 repository and include an entry for their project. Project creation is concluded upon acceptance of the pull
 request.
 
+Upon completion of the project creation process, TC Chair sends an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community)
+in order to announce the creation of the project to broader community for increased visibility.
```
Comment (d-stahl-ericsson) [inline PROJECT_LIFECYCLE.md:68]:
```suggestion
in order to announce the creation of the project to the broader community for increased visibility.
```

File: PROJECT_LIFECYCLE.md [hunk 3]
```diff
@@ -77,6 +87,9 @@ stage listed on the project repositories, project maintainers are expected to up
 a pull request to community repository, updating the stage of their project for broader visibility. Project review
 is concluded upon acceptance of the pull request.
 
+Upon completion of the project lifecycle process, TC Chair sends an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community)
+in order to announce the changes to project lifecycle stage to broader community for increased visibility.
```
Comment (d-stahl-ericsson) [inline PROJECT_LIFECYCLE.md:91]:
```suggestion
in order to announce the changes to project lifecycle stage to the broader community for increased visibility.
```

**Review by d-stahl-ericsson — CHANGES_REQUESTED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Review by d-stahl-ericsson — APPROVED** [review]

**Review by henning-roos — APPROVED** [review]

File: PROJECT_LIFECYCLE.md [hunk 4]
```diff
@@ -38,12 +38,19 @@ their ability.
...
+In addition to formally proposing the project on GitHub, community members proposing the new project are expected
+to announce the project proposal on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) by
+sending an email in order to increase the visibility. The email should contain the same information as the proposal
+itself.
+
 Regarding project naming conventions, the Eiffel Community enforces no name convention but meaningful descriptive
 names are encouraged.
 
 Projects that are already hosted within Eiffel Community but willing to move to a different stage are also
 expected to follow same process as project creation by formally submitting their request on GitHub Eiffel
```
Comment (fredjn) [inline PROJECT_LIFECYCLE.md:50]: remove GitHub
```suggestion
expected to follow same process as project creation by formally submitting their request on Eiffel
```

**Review by fredjn — CHANGES_REQUESTED** [review]

**Review by fredjn — APPROVED** [review]

**Pull request merged**